util/maven/xml_formatting.py

```diff
@@ -91,10 +93,16 @@ def maven_dependency_xml(artifact_string):
 def generate_pom(artifact_string, metadata, deps, version):
   group, artifact, version = artifact_string.split(':')
 
+  manual_deps = metadata.get('manual_dependencies', '')
+  if manual_deps:
```

Comment:
Not really, it's just to prevent doing extra work when not needed (in a previous version –maybe only locally–, it also prevented generating an extra line feed when there was no `manual_dependencies`)